src/components/card/examples/card-image/index.njk

```diff
@@ -26,9 +24,7 @@
                 "url": '#0',
                 "text": 'For Census 2021, we’ll be hiring at least 30,000 field staff across England and Wales.',
                 "image": {
-                    "smallSrc": '/img/small/placeholder-card.png',
-                    "largeSrc": '/img/large/placeholder-card.png',
-                    "alt": 'Image placeholder'
+                    "placeholderURL": ''
```

Comment:
@jrbarnes9 do we need to pass in an empty string here?

Reply:
I've reverted the macro changes I made to allow for a placeholder without setting an image, and raised a new issue (#2238) to improve this separately